tests/storage/hotplug.go

```diff
@@ -240,6 +261,33 @@ var _ = SIGDescribe("Hotplug", func() {
 		}, 360*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
 	}
 
+	verifyNoVolumeAttached := func(vmi *v1.VirtualMachineInstance, volumeNames ...string) {
+		By("Verify that the number of attached volumes does not change")
+		nameMap := make(map[string]bool)
```

Comment:
Would change to `volumeNamesMap` to be clearer

Reply:
Also, since size of map is known on creation time, we can `make(map[string]bool, len(volumeNames))`

Reply:
Also, the Go idiomatic way to check if a key exists without the value being important is to have `map[string]interface{}`. Then, to check existence you can `_, exists := myMap[key]`. This is not critical at all, so your call if to change it or not :)

Reply:
Thanks so much, I will rename and add the size of the map on creation time. :slightly_smiling_face: 


> Also, the Go idiomatic way to check if a key exists without the value being important is to have `map[string]interface{}`. Then, to check existence you can `_, exists := myMap[key]`. This is not critical at all, so your call if to change it or not :)

Maybe I'm missing something, it is not exactly what is done below?
`if _, ok := nameMap[volumeStatus.Name]; ok && ...`

Reply:
Yes :) but when defining `map[string]bool` we have to allocate space for each value in this map, which is a boolean. When you do something like `if _, ok := nameMap[volumeStatus.Name]; ok && ...` you're dismissing the value - but it's still there - every key has a bool value allocated to it.

This is why in such circumstances it's better to have `map[string]interface{}`, since as opposing to bool, interface{} demands no extra space to be allocated.

This is of course (on maps that are small enough, which is definitely the case here) negligible, so you don't have to change it, just wanted to raise your awareness to it :)

Reply:
Ah ok, got you :) Thank you, great to learn something. I'll do it, so if someone in the future get ideas from this code for something, they will do it in the best way  :)

Reply:
@fossedihelm I'm sorry, but it seems I made a mistake here :sweat_smile: 
It's actually `map[string]struct{}`, not `map[string]interface{}`. While `interface{}` is eventually a pointer to an unknown type (similar to C++'s `void*`), `struct{}` means there's no value at all.

BTW, memory usage is not the only motivation to use this, but also when using `map[string]struct{}` it's absolutely clear that the maps checks for key existence only and that the value doesn't matter in any way.

Sorry for misleading you. I also learned something new from this conversation, so thank you :)

Reply:
@iholder-redhat Not a problem at all :) Great to learn something together ;) I pushed the changes. Thanks so much